Cargo.toml

```diff
@@ -14,8 +14,8 @@ description = "A binary serialization / deserialization strategy that uses Serde
 [dependencies]
 byteorder = "1.0.0"
 num-traits = "0.1.32"
-serde = { git = "https://github.com/serde-rs/serde", branch = "master" }
+serde = "1.0.*"
```

Comment:
This disallows upgrading to 1.1.0 which is not a breaking change. I think this should say "1.0".